src/main/webapp/js/webrtc_adaptor.js

```diff
@@ -349,11 +352,13 @@ function WebRTCAdaptor(initialValues)
 			track.onended = null;
 			track.stop();
 		});
+		
+		
 
 	}
 
 	/**
-	 * Checks browser supports screen share feature
+	 * Checks broenwser supports screen share feature
```

Comment:
there is no need to change this comment line

Reply:
Ok. It's fixed. Take a look at the latest commit